configs/config.yml

```diff
@@ -0,0 +1,34 @@
+log:
+  level: debug
+  pretty: true
+
+poller:
+  enabled: true
+  interval: 3000
+  blocks_per_poll: 3
+  from_block: 20000000
+  until_block: 20000010
+
+commiter:
+  enabled: true
+  interval: 10000
+  batch_size: 100
+
+failure_recoverer:
+  enabled: true
+  interval: 3000
+  batch_size: 100
+
+storage:
+  main:
+    driver: "clickhouse"
```

Comment:
do we have to specify the name of the driver? we already have the name of the driver in the next line

Reply:
I removed it and pushed the change. Now the side-effect of this is that if the driver does not require configuration (like the memory one, if you don't want to configure maxItems), then you cannot leave it blank and have to configure it